src/components/HeroSection.astro

```diff
@@ -1,8 +1,8 @@
 ---
 import Container from "./Container.astro"
-import { Picture } from '@astrojs/image/components';
```

Comment:
the componenet ``picture`` was deprecated. will have to be fixed, see docs [here](https://docs.astro.build/en/guides/images/#remove-astrojsimage).

i can update this later if needed